### PR TITLE
[PyTorch] Add finetune

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1526/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1526/

**Comment by AnirudhDagar** [comment]
@astonzhang note that `lr` used for pytorch is orders of magnitude lower than the same used for mxnet. This is necessary for stable loss plot. Also the pretrained model available in torchvision is `resnet18` unlike `resnet18_v2` in gluon.

**Comment by astonzhang** [comment]
> @astonzhang note that `lr` used for pytorch is orders of magnitude lower than the same used for mxnet. This is necessary for stable loss plot. Also the pretrained model available in torchvision is `resnet18` unlike `resnet18_v2` in gluon.

Thanks for the note.
